src/ctelnet.cpp

```diff
@@ -1941,7 +1941,7 @@ void cTelnet::setGMCPVariables(const QByteArray& msg)
         data = transcodedMsg.section(QChar::LineFeed, 1);
     }
 
-    if (transcodedMsg.startsWith(QStringLiteral("Client.GUI"))) {
+    if (transcodedMsg.startsWith(QStringLiteral("Client.GUI"), Qt::CaseInsensitive)) {
```

Comment:
BTW comparing with the next change it seems that it is valid to use `QLatin1String("Client.GUI")` here and for the previous one - and the compiler can combine them into a common symbol...

Reply:
Sure, but a QStringLiteral is faster because it's made at compile time.

Reply:
Okay, well define a common e.g. `const static QString CLIENT_GUI{QStringLiteral("Client.GUI")};` instance at the top of the file to hold it for as many (well any ***more*** than **one**) uses in this *compilation unit*...

Reply:
No thank you, don't like this trend. More harm than help...